src/platforms/common/performance/instrumentation/custom-instrumentation.mdx

```diff
@@ -33,84 +33,106 @@ redirect_from:
 
 <Note>
 
-To capture transactions and spans customized to your organization's needs, you must first <PlatformLink to="/performance/">set up performance monitoring.</PlatformLink>
+This is just a THOUGHT experiment and should not be merged!
 
 </Note>
 
-<PlatformSection notSupported={["javascript", "node"]}>
+## Creating Spans
 
-<PlatformContent includePath="performance/enable-manual-instrumentation" />
+```javascript
+function doWork() {
+  const result = Sentry.startSpan({ name: "Important Function" }, () => {
```

Comment:
I guess the struggle with naming here is between `startActiveSpan` vs. `startSpan`.

I can defer to whatever the majority thinks should be done but here is some of my reasoning toward choosing the existing naming in the RFC.

1. It aligns with OpenTelemetry
2. `Sentry.startSpan` maps in behavior exactly to `Sentry.startTransaction` (doesn't add itself to scope, returns a span/transaction to use, etc.)
3. `startActiveSpan` indicates something special happening to the passed in callback. `trace` is also good here, but is an overloaded word.

If we wanted to explore alternative naming, `withSpan` or `withActiveSpan` also makes sense in my head.